src/vs/workbench/common/backup.ts

```diff
@@ -0,0 +1,119 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
:+1: not sure whether service I copied but it must have been shuffled within the last 3 weeks.